scripts/generate_upload_sbom.sh

```diff
@@ -95,16 +96,31 @@ echo "Platforms:" "${platforms[@]}"
 echo "S3 Path: $s3_path"
 
 for platform in "${platforms[@]}"; do
-  s3_path_platform_dependent="$s3_path/${platform////_}"
-  file_name="${image_name}_${tag_name}_${platform////_}.json"
+  os=${platform%/*}
+  arch=${platform#*/}
+
+  s3_path_platform_dependent="$s3_path/${os}_${arch}"
+  file_name="${image_name}_${tag_name}_${os}_${arch}.json"
```

Comment:
I believe we now put it in the right folder, but the name will not match what the checklist doc links expect `${os}-${arch}.sbom.json`